examples/terraform-submodule/module.tf

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Run:
+//  terraform apply -var project="<YOUR_GCP_ProjectID>" -var agones_version="0.9.0"
+variable "password" {
+   default= ""
+}
+variable "agones_version" {
+  default="0.9.0"
```

Comment:
I'm just hesitant to set the version as default. I already have a long list of things to update on release - this just adds another one. Can we use the helm default of installing the latest, with the ability to override?